flocker/node/_loop.py

```diff
@@ -0,0 +1,100 @@
+"""
+Convergence loop for the agent.
```

Comment:
Probably worth saying "dataset" at least once somewhere here since this is only the (zfs?) dataset convergence loop.  Although most of this code looks like it is generic and applicable to any of our convergence agents.  I imagine we'll refactor as soon as we want a second kind of agent.